liquidctl/driver/aura_led.py

```diff
@@ -0,0 +1,347 @@
+        else:
+            status.append("Unexpected reply for firmware", "", "")
+            return status
+
+        # This stops Direct mode if it was previously applied
+        self._write(_FUNCTION_CODE["end_direct"])
+
+        """
+        Extra operations during initialization
+        This is experimental and may not be necessary
+        self._write([0xec, 0x31, 0x0d, 0x00]);
+        self._write([0xec, 0xb1, 0x00, 0x00]);
+        self.device.read(_READ_LENGTH)
+        self._write([0xec, 0x31, 0x0e, 0x00]);
+        self._write([0xec, 0xb1, 0x00, 0x00]);
+        self.device.read(_READ_LENGTH)
+        self._write([0xec, 0x31, 0x0f, 0x00]);
+        self._write([0xec, 0xb1, 0x00, 0x00]);
+        self.device.read(_READ_LENGTH)
+        self._write([0xec, 0x83, 0x00, 0x00]);
+        self.device.read(_READ_LENGTH)
+        """
+        return status
+
+    def get_status(self, **kwargs):
+        """Get a status report."""
+        status = []
+        # Get config table
+        self._write(_FUNCTION_CODE["config"])
+        data = self.device.read(_READ_LENGTH)
+        if data[1] == 0x30:
+            start_index = 4  # index of first record
+
+            argb_channels = data[start_index + 2]
+            rgb_channels = data[start_index + 3]
+            status.append(("ARGB channels: " + str(argb_channels), "", ""))
+            status.append((" RGB channels: " + str(rgb_channels), "", ""))
+
+            if "debug" in kwargs and kwargs["debug"] == True:
+                num = 6  # number of bytes per record
+                count = 1
+                while start_index + num < _READ_LENGTH:
+                    status.append(
+                        (
+                            "Device Config: " + str(count),
+                            ", ".join(
+                                "0x{:02x}".format(x) for x in data[start_index : start_index + num]
+                            ),
+                            "",
+                        )
+                    )
+                    start_index += num
+                    count += 1
+        else:
+            status.append("Unexpected reply for config", "", "")
+        return status
+
+    def set_color(self, channel, mode, colors, speed="normal", **kwargs):
+        """Set the color mode for a specific channel.
+
+        `colors` should be an iterable of zero or one `[red, green, blue]`
+        triples, where each red/green/blue component is a value in the range
+        0–255.
+        """
+        colors = iter(colors)
+        if _COLOR_MODES[mode].takes_color:
+            try:
+                r, g, b = next(colors)
+                single_color = (r, g, b)
+            except:
+                raise ValueError(f"one color required for this mode") from None
+        else:
+            single_color = (0, 0, 0)
+
+        if channel != "sync" and channel not in _COLOR_CHANNELS:
+            # _LOGGER.error("channel %s not valid", channel)
```

Comment:
Please remove the unused debugging code. (I think thats what these commented out `_LOGGER` lines are.

Reply:
Lines have been removed.